lib/utils/import.js

```diff
@@ -49,3 +50,33 @@ function getSourceModuleName(node) {
     return undefined;
   }
 }
+
+/**
+ * Gets a list of import identifiers from the source node. Will return the imported
+ * identifiers' imported name if no alias is used, or local name if an alias is used.
+ *
+ * @example
+ *
+ * getImportIdentifiers(node, {
+ *  '@ember/object': ['set', 'setProperties'],
+ * });
+ *
+ * @param {*} node
+ * @param {*} imports
+ * @returns
+ */
+function getImportIdentifiers(node, imports) {
```

Comment:
I'm wondering if we could simplify the API of this function. Most of the time, we just want to get the identifier that a single function is imported under. So what if we replaced the object parameter with a `path` argument and an optional `exportName` argument (to support default exports)?

```js
getImportIdentifier(node, path, namedExport = null) { ... }
```

In the case of this particular rule, we could call the function a few times to collect the identifiers we need.

```js
const testWaitersIdentifiers = LEGACY_TEST_WAITER_FUNCTIONS.map(fn => getImportIdentifier(node, '@ember/test', fn));
```

Some more examples:

```js
// import Component from '@ember/component';
getImportIdentifier(node, '@ember/component') // returns 'Component'
```

```js
// import { empty } from '@ember/object/computed';
getImportIdentifier(node, '@ember/object/computed', 'empty') // returns 'empty'
```

```js
// import { inject as service } from '@ember/service';
getImportIdentifier(node, '@ember/service', 'inject') // returns 'service'
```

Reply:
I like this. Simpler. Doing it.